tools/ts-generator/rpc/rpc_gen_js.py

```diff
@@ -79,72 +82,135 @@
     RpcHeader.toBytes(rpc, reserve);
 }
 
+const startReadRequest = (fn: ApplyFn) => (socket: Socket): void => {
+  socket.once("data", (data) => readRequestBuffer(data, fn, socket));
```

Comment:
`readBufferRequest`*

Also, let's add a comment about what this does.

Reply:
Done